app/presenters/single_content_item_presenter.rb

```diff
@@ -25,6 +25,11 @@ def parse_time_series(time_series)
     self
   end
 
+  def set_date_range(date_range)
```

Comment:
See my comment above, but I would remove from the public API of this class the methods:
- set_date_range
- parse_metrics
- parse_time_series